diff --git a/bazel/gflags.bzl b/bazel/gflags.bzl
index 4199e31..176fcb3 100644
--- a/bazel/gflags.bzl
+++ b/bazel/gflags.bzl
@@ -77,6 +77,8 @@ def gflags_library(hdrs = [], srcs = [], threads = 1):
         "-DHAVE_STRTOLL",
         "-DHAVE_STRTOQ",
         "-DHAVE_RWLOCK",
+        "-Wno-unused-parameter",
+        "-Wno-cast-function-type",
     ] + select({
         "//:x64_windows": [
             "-DOS_WINDOWS",
